etc/mpv.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>io.mpv.Mpv</id>
```

Comment:
where does this id come from?

Reply:
https://github.com/flathub/io.mpv.Mpv

Reply:
https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic

Reply:
Hmm okay. I think `io.mpv.mpv` would be better in this case.

Reply:
Fixed.